lib/components/app/app-menu.tsx

```diff
@@ -154,6 +154,7 @@ class AppMenu extends Component<
             role={isSelected !== undefined ? 'option' : undefined}
             subItems={this._addExtraMenuItems(children) || undefined}
             text={label}
+            title={typeof label === 'string' ? label : ''}
```

Comment:
Actually, could you add this check inside of `AppMenuItem` and apply it to the `text` prop, so that if `text` is a string and `title` is not specified, then use `text` as the title. This would reduce the duplicate values passed to `AppMenuItem` below.